.travis.yml

```diff
@@ -1,50 +0,0 @@
-language: php
-os: linux
-dist: xenial
-
-services:
-  - mysql
-
-notifications:
-  email:
-    on_success: never
-    on_failure: change
-
-php:
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
-  - 7.3
-  - 7.4
-  - 8.0
-  - "nightly"
```

Comment:
One thing I've just realised is missing from the new GitHub actions, is testing against PHP nightly. Currently, PHP 8.1 is on an [alpha 2 release](https://www.php.net/archive/2021.php#2021-06-24-1), so testing against it and not blocking the workflow if it fails, would be a nice way to get ahead of the curve, especially as we work towards adding unit and integration tests.